events-api-reference.yaml

```diff
@@ -0,0 +1,1070 @@
+swagger: '2.0'
+info:
+  title: I/O Management API
+  version: '2.0'
+host: api-stage.adobe.io
```

Comment:
so this is for stage ? is that on purpose ?

Reply:
This came from the file that was provided: https://git.corp.adobe.com/pages/adobe-apis/api-mgmt-docs/#/events

The base is shown at the link above as `api-stage.adobe.io` and then when you click the link in the swagger example to view the json, that is the host url.

Reply:
yes prod is also available at https://git.corp.adobe.com/pages/adobe-apis/api-mgmt-docs/#/ see the `select a spec`drop down 
the associated file is there : https://git.corp.adobe.com/adobe-apis/api-mgmt-docs/blob/master/v2/api-docs-prod.json